pages/faqs.vue

```diff
@@ -2,10 +2,50 @@
 definePageMeta({
   layout: 'landing',
 })
+
+interface Faq {
+  text: string
+  link: string
+}
+
+const faqs = [
+  {
+    text: 'How do I reset my account password?',
+    link: '/login',
+  },
+  {
+    text: 'How do I turn off notifications?',
+    link: '',
+  },
+  {
+    text: 'How do I upload a file to the forum?',
+    link: '',
+  },
+  {
+    text: 'How do I create an event for a group?',
+    link: '',
+  },
+  {
+    text: 'How do I redeem reward vouchers?',
+    link: '',
+  },
+  {
+    text: 'How do I reset my account password?',
+    link: '',
+  },
+] satisfies Faq[]
 </script>
 
 <template>
-  <div>
-    FAQ page
+  <div flex flex-col items-center>
+    <h1 p-5 text-center text-3xl>
+      Frequently Asked Questions
+    </h1>
+    <div>
+      Search Bar Here
+    </div>
+    <div class="w-full md:w-3/5" grid grid-cols-1 mt-5 gap-3 overflow-hidden border-2 border-outline rounded-3xl pb-3 text-left divide-y-2 divide-outline>
+      <FAQRedirect v-for="faq in faqs" v-bind="faq" :key="faq.text" />
```

Comment:
I think it might not be worth putting a separate component for this considering it is only 1 NuxtLink comp. Thoughts?

Reply:
I was thinking about that but I mean if we keep linking it may be more efficient. no?

Reply:
Hm, I meant taking out FAQRedirect and using NuxtLink directly.

Example:

```diff
-     <FAQRedirect v-for="faq in faqs" v-bind="faq" :key="faq.text" />
+     <NuxtLink px-3 pt-3 v-for="faq in faqs" :key="faq.text" :to="faq.to">
+         {{ props.text }}
+     </NuxtLink>
```